runtime/error.c

```diff
@@ -116,13 +129,13 @@ static int storePC(void *data, PC pc) {
     return SUCCESS;
 }
 
-void _bal_error_backtrace_print(ErrorPtr ep) {
-    GC PC *pcs = ep->pcs;
+void _bal_error_backtrace_print(ErrorPtr ep, uint32_t start) {
```

Comment:
I would add a FILE * as a parameter here, so you can test it more easily.

Reply:
Fixed